addOns/commonlib/commonlib.gradle.kts

```diff
@@ -0,0 +1,128 @@
+import org.zaproxy.gradle.addon.AddOnStatus
+
+plugins {
+    `maven-publish`
+    signing
+}
+
+group = "org.zaproxy.addon"
+
+version = "0.1.0"
+description = "A common library, for use by other add-ons."
+
+zapAddOn {
+    addOnName.set("Common Library")
+    addOnStatus.set(AddOnStatus.ALPHA)
+    zapVersion.set("2.9.0")
+
+    manifest {
+        author.set("ZAP Dev Team")
+
+        helpSet {
+            baseName.set("help%LC%.helpset")
+            localeToken.set("%LC%")
+        }
+    }
+}
+
+dependencies {
+    zap("org.zaproxy:zap:2.7.0")
```

Comment:
This is using the version that was in the `sharedutils` project, should we bump to 2.9.0?

Reply:
I think that was just a timing thing. I'm fine with going to 2.9.0